Dockerfile

```diff
@@ -1,18 +1,16 @@
-FROM python:3.9.1-buster
+# leverage the renci python base image
+FROM renciorg/renci-python-image:v0.0.1
 
+# switch to the new user created in the above image
+USER nru
 
 # set up requirements
-ADD ./requirements.txt ./requirements.txt
+COPY ./requirements.txt ./requirements.txt
 RUN pip install -r ./requirements.txt
 
-# create a new user and use it.
-RUN useradd -M -u 1001 nonrootuser
-USER nonrootuser
 # set up source
-ADD ./app ./app
-ADD ./main.sh ./main.sh
+COPY . .
```

Comment:
Can we just do this at line 8 instead of line 8?

Reply:
The way it is now, Docker will cache the python installation step and make subsequent builds much quicker. This speeds development up when you're making changes to just the code.